### PR TITLE
Added 11 digit restriction for toll free numbers

### LINKED ISSUE #4289: Modify customer care number condition in asset creation page
**Describe the bug**

Currently, we are only able to input 10-digit numbers in the customer care column of the asset creation page, but most of the toll-free numbers are 11 digit

**To Reproduce**

https://care.coronasafe.in/facility/4f83b5ff-7683-499f-b330-3f49b85fef23/assets/52270b31-9248-4e6e-8e5d-350f179c078d

**Expected behavior**

We should be able to input numbers up to 11 digits in the customer care column

**Screenshots**

image

### PR DESCRIPTION
**Pull request by skks1212**
## Proposed Changes

- Fixes #4289 
- Allows 11 digits only if they start from `1800`

## Backend Changes Required

- Waiting for [https://github.com/coronasafe/care/pull/1121](https://github.com/coronasafe/care/pull/1121) to be merged.

@coronasafe/care-fe-code-reviewers @coronasafe/code-reviewers

## Merge Checklist

- [ ] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.coronasafe.network/coronasafe-care-documentation/architecture/architecture-and-layering-of-care).
- [ ] Ensure that UI text is kept in I18n files.
- [ ] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [ ] Request for Peer Reviews
- [ ] Completion of QA

### CODE REVIEW

**Comment by nihal467** [comment]
@skks1212 Currently, there is no limit for the number on the platform, which doesn't look good. 

is it possible only for the customer care number inside the asset creation form to have 11 number restrictions? and all other phone numbers remain to have 10 number restriction 

 @gigincg @khavinshankar what are your opnion

**Comment by khavinshankar** [comment]
@nihal467 @skks1212 I agree, may be on top of it we could have some kind of isValidPhoneNumber regex or function to validate it :thinking:

**Comment by shivankacker** [comment]
@nihal467 @gigincg @khavinshankar I have now only allowed 11 digits to be entered if the number starts with `1800`

**Comment by nihal467** [comment]
@skks1212 
image

https://deploy-preview-4294--care-net.netlify.app/facility/4f83b5ff-7683-499f-b330-3f49b85fef23/assets/52270b31-9248-4e6e-8e5d-350f179c078d

I am not able to submit the asset form

**Comment by shivankacker** [comment]
@nihal467 will have to create a backend PR to handle backend validation. Marking PR as waiting

**Comment by nihal467** [comment]
@skks1212 
image

still not working

File: src/Components/Facility/AssetCreate.tsx [hunk 1]
```diff
@@ -775,6 +785,7 @@ const AssetCreate = (props: AssetProps) => {
                       Customer Support Number *{" "}
                     </label>
                     <PhoneNumberField
+                      enableTollFree
```
Comment (khavinshankar) [inline src/Components/Facility/AssetCreate.tsx:788]: @skks1212 I think you missed to add `countryCodeEditable` prop here
```suggestion
                      enableTollFree
                      countryCodeEditable
```

**Comment by khavinshankar** [comment]
@skks1212 can you check this and clear it out as soon as possible

File: src/Components/Facility/AssetCreate.tsx (hunk 1 above)
Comment (shivankacker) [reply src/Components/Facility/AssetCreate.tsx:788]: Nope! There is no need for editing the country code.

**Comment by nihal467** [comment]
@skks1212 the PR is not working 
image

**Comment by shivankacker** [comment]
@khavinshankar @nihal467 There appears to be a problem with care backend not deploying. Im adding a waiting for backend label to the PR again.

**Comment by shivankacker** [comment]
@nihal467 
image
Looks like Deploy issue has been fixed. Can you try testing again? 🥲

**Comment by nihal467** [comment]
@skks1212 the PR is still failing

**Comment by nihal467** [comment]
@skks1212 https://app.flonnect.com/view/video/nihalcct/639ea4a6f31aaa756ef93aa9

when we are typing 18001801234, there is a bouncing happening after the number 1800,

**Comment by nihal467** [comment]
@khavinshankar 
image

now the normal phone number has become 11 digit and it is not the solution we discussed

**Comment by khavinshankar** [comment]
@nihal467 merging this pr as it is a p1 issue, following this issue in a new pr

**Comment by nihal467** [comment]
@khavinshankar @skks1212  conditionally approving this solution, as it is temporarily solving purpose, but the improved solution needs to be pushed before next week's release.

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**